.travis.yml

```diff
@@ -1,10 +1,12 @@
 before_install:
+  - gem install bundler
   - gem update bundler
```

Comment:
Apparently, we need `gem install bundler` because Travis does not install bundler for jruby and we also need `gem update bundler` because the bundler version installed for other Ruby implementations is obsolete. So let's keep both...